Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -2443,8 +2463,23 @@ def upload_XSOAR_attachment_to_jira(client: JiraBaseClient, entry_id: str,
         List[Dict[str, Any]]: The results of the API, which will hold the newly added attachment.
     """
     file_name, file_bytes = get_file_name_and_content(entry_id=entry_id)
-    files = {'file': (attachment_name or file_name, file_bytes, 'application-type')}
-    return client.upload_attachment(issue_id_or_key=issue_id_or_key, files=files)
+    files = create_files_to_upload('', file_name, file_bytes, attachment_name)
+    # Index 2 holds the MIME type
+    chosen_file_mime_type = files.get("file", ())[2]
```

Comment:
lets instead of this get, return both the file dict and the chosen_file_mime_type from the method.

`return {'file': (attachment_name or file_name, file_bytes, mime_type_to_upload)}, mime_type_to_upload`